indicators/banco-de-galicia-2mo4sf.yml

```diff
@@ -0,0 +1,29 @@
+title: Banco de Galicia Phishing Kit 2mO4SF
+description: |
+    Detects a phishing kit targeting Banco de Galicia. The threat actor operates from Argentina itself.
+
+
+references:
+    - https://urlscan.io/result/d432c567-d1de-4b56-9ddb-af47248ab423/
+
+detection:
+
+    malformedHTML:
+      html|contains:
+        - <img src="img/109.jpg" width="100%" <title="">
+
+    banner:
+      html|contains:
```

Comment:
A better indicator might be the CSRF token `NOkPDuCJE_NXUBMT9OQG-KhVO3-R8Uruo-giLM1tLPZAsdDMuxuWaFsp-kpbIt1CHFjZx6z644GVwxBiB9gp6U0zaZWoM_pAGYsQZLEfUM01`